### ISSUE TITLE
logistic regression -sensitivity and specificity values

### ISSUE THREAD

**Issue by marutimishra** [jasp-stats/jasp-issues#707]
* JASP version:0.11.1

* OS name and version: windows
* Analysis: Logistic regression
* Bug description: The performance metrics show the results for the sensitivity and specificity of the test. Is it possible that the values are flipped? When I do a ROC analysis in SPSS and get a value for the same, the results seem flipped. I am not sure if its SPSS or JASP, but I do not get the same results
* Expected behaviour: Is it possible to confirm this behaviour somehow?

**Comment by boutinb** [comment]
@vandenman can you have a look to this issue?

**Comment by fqixiang** [comment]
I looked into this issue. The R code used to calculate sensitivity and specificity in JASP is correct (which is based on the HMeasure function from an existing library called hmeasure). I also ran a logistic regression model myself on a data set I am familiar with; the output sensitivity and specificity scores are also correct. So I don't think there is something to be fixed on the JASP part. The most likely reason why the sensitivity and specificity scores are flipped in SPSS is that the outcome binary variable is encoded in the opposite way in SPSS (by accident or default). 
I would therefore suggest closing this issue (also taking into account that @marutimishra did not include a reproducible example, which makes it difficult to look into the specific cause of the flipped scores).
